src/app/pages/mis-eventos/mis-eventos.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { Evento } from 'src/app/models/evento';
+import { EventoService } from '../../services/evento.service';
+import { Router } from '@angular/router';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138913)
```suggestion
import { Router } from "@angular/router";
```